### ISSUE TITLE
Real math function unittest failure for SLES11.4 build

### ISSUE THREAD

**Issue by jrudess** [MikePopoloski/slang#434]
I'm working on deploying slang in a corporate environment using SLES11.4, and the build is successful but one unit test fails.
```
slang/tests/unittests/EvalTests.cpp:1522: FAILED:
  CHECK( session.eval("$tan(123.456)").real() == tan(123.456) )
with expansion:
  1.35181 == 1.3518050822
```
Tool versions are:
```
cmake 3.15.2
gcc 9.2
python 3.6
libc 2.11.3
```
gcc or libc seems like the most likely culprit, but wanted to check if you have any other suggestions.

**Comment by MikePopoloski** [comment]
Yeah, I assume gcc and libc are compiled with differing rounding modes or something here. I can change these tests to check for equality within epsilon to make them more stable.

**Comment by MikePopoloski** [comment]
Fixed in c5b971818bb8b54935c40a5bb54432764e5f1ce4
